AerospikeClient/Async/AsyncCommand.cs

```diff
@@ -203,14 +204,25 @@ private void ExecuteCore()
 			}
 			ExecuteCommand();
 		}
-
+		
 		private void ExecuteCommand()
 		{
 			iteration++;
+			
+			Exception exception = null;
+
+			StartActivity();
 
 			try
 			{
 				node = (AsyncNode)GetNode(cluster);
+				
+				if (activity != null)
+				{
+					activity.SetTag("node.name", node.Name);
+					activity.SetTag("node.address", node.NodeAddress.ToString());
+				}
```

Comment:
Those don't seem to be standard tags, I would use `network.peer*` as mentioned below.